src/index.js

```diff
@@ -0,0 +1,19 @@
+// import _ from 'lodash';
+import './style.css';
+// import printMe from './print.js';
+import ScoresList from './modules/ScoresList';
```

Comment:
[OPTIONAL] I think it's better to just remove comments from the codebase, and the same applies to files or dependencies that we might not be using. That's because usually, we want our codebase as clean and professional as possible since we're working either with a team or collaboratively on an open-source project.